logger.js

```diff
@@ -65,6 +65,16 @@ Logger.prototype.error = function error(err) {
             fullType: err.fullType
         }));
 
+    } else if (err.type === 'tchannel.request.timeout') {
```

Comment:
This line will take the error frame case:
https://github.com/uber/tcurl/pull/98/files#diff-7f28009834c7b72b5b0080bc0487ccddL45